x-pack/plugin/searchable-snapshots/src/internalClusterTest/java/org/elasticsearch/xpack/searchablesnapshots/FrozenSearchableSnapshotsIntegTests.java

```diff
@@ -263,22 +253,6 @@ public void testCreateAndRestorePartialSearchableSnapshot() throws Exception {
             // the original shard size from the snapshot
             final long originalSize = snapshotShards.get(shardRouting.getId()).getStats().getTotalSize();
             totalExpectedSize += originalSize;
-
-            // an extra segments_N file is created for bootstrapping new history and associating translog. We can extract the size of this
-            // extra file but we have to unwrap the in-memory directory first.
-            final Directory unwrappedDir = FilterDirectory.unwrap(
-                internalCluster().getInstance(IndicesService.class, getDiscoveryNodes().resolveNode(shardRouting.currentNodeId()).getName())
-                    .indexServiceSafe(shardRouting.index())
-                    .getShard(shardRouting.getId())
-                    .store()
-                    .directory()
-            );
-            assertThat(shardRouting.toString(), unwrappedDir, notNullValue());
-            assertThat(shardRouting.toString(), unwrappedDir, instanceOf(ByteBuffersDirectory.class));
-
-            final ByteBuffersDirectory inMemoryDir = (ByteBuffersDirectory) unwrappedDir;
-            assertThat(inMemoryDir.listAll(), arrayWithSize(1));
```

Comment:
Could we leave this in place but verify there are no files modified - at least until we possibly remove the in-memory-no-op-commit-directory?

Reply:
Makes sense yes, that's a good reason to keep this.